public/404.html

```diff
@@ -1,67 +0,0 @@
-<!DOCTYPE html>
```

Comment:
@danielpaul The other guys also said to delete these public error html files since we already created the dynamic ones.